src/el-data-table.vue

```diff
@@ -6,7 +6,7 @@
             <slot name="search"></slot>
             <el-form-item>
                 <!--https://github.com/ElemeFE/element/pull/5920-->
-                <el-button native-type="submit" type="primary" @click="page = firstPage; getList(1)" size="small">查询</el-button>
+                <el-button native-type="submit" type="primary" @click="page = defaultFirstPage; getList(defaultFirstPage)" size="small">查询</el-button>
```

Comment:
getList(defaultFirstPage), this function is unclear.
see  [https://github.com/ryanmcdermott/clean-code-javascript#dont-use-flags-as-function-parameters](url)
```suggestion
                <el-button native-type="submit" type="primary" @click="page = defaultFirstPage; getList(defaultFirstPage)" size="small">查询</el-button>
```
```suggestion
                <el-button native-type="submit" type="primary" @click="page = defaultFirstPage; getList(defaultFirstPage)" size="small">查询</el-button>
```
the function getList(defaultFirstPage) is unclear.
see [https://github.com/ryanmcdermott/clean-code-javascript#dont-use-flags-as-function-parameters](url)
try to refactor it, make code clean as possible as you can